cpp/daal/src/externals/service_math.h

```diff
@@ -50,18 +50,32 @@ struct Math
 
     static fpType sPowx(fpType in, fpType in1) { return _impl<fpType, cpu>::sPowx(in, in1); }
 
+    static fpType xsPowx(fpType in, fpType in1) { return _impl<fpType, cpu>::xsPowx(in, in1); }
```

Comment:
Could you please leave some comments in this header about the differences between the functions starting with 's' and the functions starting with 'xs' and when should each be called.